themes/website/templates/download.html

```diff
@@ -49,10 +50,11 @@ <h1 class="header">Thanks for downloading!</h1>
 				<div class="col-sm-4 feature">
 					<img src="theme/img/plat/linux_dark.svg">
 					<h2>Linux</h2>
-					<p class="lead"><a href="#gnulinux">Repo (Apt, Gentoo, Arch)</a></p>
-					<p class="lead" style="margin-top:-15px;"><a href="#gnulinux">qTox package</a></p>
-					<p class="lead" style="margin-top:-15px;"><a href="https://build.tox.chat/job/uTox_build_linux_x86_release/lastSuccessfulBuild/artifact/utox_linux_x86.tar.xz">uTox 32 bit</a> | <a href="https://build.tox.chat/job/uTox_build_linux_x86-64_release/lastSuccessfulBuild/artifact/utox_linux_x86-64.tar.xz">64 bit</a></p>
-					<p class="lead" style="margin-top:-15px;"><a href="#gnulinux">Toxic package</a>|<a href="https://build.tox.chat/job/toxic_build_linux_x86_release/lastSuccessfulBuild/artifact/toxic_build_linux_x86_release.tar.xz">32 bit binary</a> | <a href="https://build.tox.chat/job/toxic_build_linux_x86-64_release/lastSuccessfulBuild/artifact/toxic_build_linux_x86-64_release.tar.xz">64 bit binary</a></p>
+					<p class="lead"><a href="#gnulinux">Ricin, Toxic, Toxygen and uTox Debian/Ubuntu repo</a></p>
+					<p class="lead" style="margin-top:-15px;"><a href="https://software.opensuse.org/download.html?project=home%3Aantonbatenev%3Atox&package=qtox">qTox repo (Arch, CentOS, Debian, Fedora, openSUSE, Ubuntu)</a></p>
+					<p class="lead" style="margin-top:-15px;"><a href="#gnulinux">Arch and Gentoo repos (low maintenance)</a></p>
```

Comment:
Could be reworded; it's not really clear that this means the repos are unmaintained.